Makefile

```diff
@@ -279,7 +293,7 @@ distclean: clean assetclean
 
 setup:
 	$(MAKE) -C tools
-	python3 fixbaserom.py
+	python3 tools/decompress_baserom.py $(VERSION)
```

Comment:
This feels redundant right now because `extract_baserom.py` already knows how to decompress yaz0 files. Is the idea that multiversion asset extraction would work directly from the uncompressed rom instead of first extracting the rom files?

Reply:
Having a decompressed rom to begin with is nice since you can use it to compare your built uncompressed rom in case it doesn't match with tools like bdiff or vbindiff

Reply:
Alright, I'm on board